status-im/status-react
# Update Chinese translations

### Summary:
Update Chinese translations

### Steps to test:
- Change device language setting to Chinese
- Launch app, confirm that Chinese is correctly reflected

status: ready
